### PR TITLE
feat: better title handling

### PR DESCRIPTION
**Pull request by lukeromanowicz**
## Description
Resolves [#414](https://github.com/aeternity/aescan/issues/414)

## Demo
No visual changes, just refactoring of page titles implementation. They should work just like before

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] My change does not require a change to the documentation.

### CODE REVIEW

File: src/app.vue [hunk 1]
```diff
@@ -92,6 +91,14 @@ if (process.client) {
...
 }
 
+useHead({
+  titleTemplate: title => {
+    return title
+      ? `${APP_TITLE_SHORT} | ${title}`
+      : APP_TITLE
+  },
+})
+
```
Comment (janmichek) [inline src/app.vue:101]:
```suggestion
useHead({
  titleTemplate: pageTitle=> {
    return pageTitle
      ? `${APP_TITLE_SHORT} | ${pageTitle}`
      : APP_TITLE
  },
})

```

**Review by janmichek — APPROVED** [review]
Excellent! 🍰  Exactly what I proposed.
Giving just a minor code suggestion if you like it

File: src/app.vue (hunk 1 above)
Comment (lukeromanowicz) [reply src/app.vue:101]: Of course, it's much better

**Review by CharisSiarampalis — APPROVED** [review]

File: src/pages/keyblocks/[id].vue [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <template>
   <Head>
-    <Title>{{ APP_TITLE_SHORT }} | Keyblock </Title>
+    <Title>Keyblock </Title>
```
Comment (michele-franchi) [inline src/pages/keyblocks/[id].vue:3]:
```suggestion
    <Title>Keyblock</Title>
```

just an extra space

**Review by michele-franchi — APPROVED** [review]
Good job!

**Pull request merged**